doc.go

```diff
@@ -3,14 +3,22 @@ Package cocoa provides interfaces to interact with groups of containers (called
 pods) backed by container orchestration services. Containers are not managed
 individually - they're managed as logical groupings of containers.
 
-The ECSPodCreator interface provides an abstraction to create pods backed by ECS
-without needing to make direct calls to the API.
+The ECSPodCreator provides an abstraction to create pods backed by ECS without
+needing to make direct calls to the API.
 
 The ECSPod is a self-contained unit that allows users to manage their pod
 without having to make direct calls to the API. It is backed by an ECSClient.
 
-The ECSClient interface provides a convenience wrapper around the ECS API. If
-the ECSPodCreator and ECSPod do not fulfill your needs, you can make calls
-directly to the ECS API instead.
+The ECSClient provides a convenience wrapper around the ECS API. If the
+ECSPodCreator and ECSPod do not fulfill your needs, you can instead make calls
+directly to the ECS API using this client.
+
+The Vault is an ancillary service for pods that supports interacting with a
+dedicated secrets management service. It conveniently integrates with pods to
+securely pass secrets into containers.
+
+The SecretsManagerClient provides a convenience wrapper around the Secrets
```

Comment:
Is this AWS's secrets manager api? Maybe we should specify if it is.

Reply:
Sure, I can mention AWS (I also did it for ECS).